FPUB thesis template/qcd.tex

```diff
@@ -0,0 +1,217 @@
+\begin{equation*}
+    \mathcal{L}_{\textsf{quarks}}=\overline{\psi}(x)\big(i\slashed{\textsf{D}}-\textsf{M}\big)\psi(x),
+\end{equation*}
+where $\textsf{M}=\textsf{Diag}\left\{m_1,\ldots,m_{N_f}\right\}$ is the diagonal quark mass matrix in flavour space\footnote{In the Standard Model, the quark mass matrix is no longer diagonal. After spontaneous symmetry breaking, the mixing between different flavoured quark masses is given by the {\sffamily CKM} matrix \cite{pdg}.}. 
+
+The dynamics of the gluon fields is described by the following construction\footnote{It is important to notice that such a construction contains not only standard kinetic terms but also interaction vertices with three gluons, which are proportional to $g$ and four gluons, proportional to $g^2$.}
+\begin{equation*}
+    \mathcal{L}_{\textsf{gluons}}=-\frac{1}{2}\textsf{Tr}\left\{F_{\mu\nu}F^{\mu\nu}\right\},
+\end{equation*}
+where the color tracing over the contraction of field strength tensors assured gauge invariance. Equivalently, one may rewrite the above expression in terms of color components as
+\begin{equation*}
+    \mathcal{L}_{\textsf{gluons}}=-\frac{1}{4}F_{\mu\nu}^aF^{a,\mu\nu},
+\end{equation*}
+valid in the fundamental representation, where $\textsf{Tr}\left\{t^at^b\right\}=\delta^{ab}/2$. Therefore, the {\sffamily QCD} Lagrangian takes the form
+\boxedeqlabel{qcd6}{
+    \mathcal{L}_{\textsf{QCD}}=\overline{\psi}(x)\big(i\slashed{\textsf{D}}-\textsf{M}\big)\psi(x)-\frac{1}{4}F_{\mu\nu}^aF^{a,\mu\nu}
+}
+
+The corresponding Yang-Mills action expressed in flat coordinates is given by
+\begin{align}\label{yangmills}
+    \textsf{S}=\int\mathrm{d}^4x\left(-\frac{1}{2}\textsf{Tr}\left\{F_{\mu\nu}F^{\mu\nu}\right\}\right),
+\end{align}
+
+\section{Field equations} 
+The variational derivatives with respect to the color spinor fields give the colored Dirac equations
+\begin{equation*}
+    (i\slashed{\textsf{D}}-\textsf{M})\psi=0,    
+\end{equation*}
+and similarly for the anti-quark fields
+\begin{equation*}
+    \overline{\psi}(i\overleftarrow{\slashed{\textsf{D}}}-\textsf{M})=0.    
+\end{equation*}
+The Euler-Lagrange equations corresponding to the gluon fields yield the Yang-Mills equations, or equivalently, colored Maxwell equations\footnote{There is an additional equation, called the Bianchi identity, which follows from the definition and properties of $F_{\mu\nu}$. It may be expressed as \cite{tong}
+\begin{equation*}
+    \textsf{D}_\mu^{\phantom{*}*}F^{\mu\nu}=0,
+\end{equation*}
+where we introduced the dual field strength tensor as
+\begin{equation*}
+    ^*F^{\mu\nu}=\frac{1}{2}\epsilon^{\mu\nu\rho\sigma}F_{\rho\sigma}.
+\end{equation*}}
+\shadedeq{
+    \textsf{D}_\nu F^{\nu\mu}=gJ^\mu
+}
+in which $J^\mu=\sum\limits_a J^{a,\mu} t^a$ with $J^{a,\mu}=\overline{\psi}\gamma^\nu t^a\psi$ being the color current. The color current is covariantly conserved $\textsf{D}_\mu J^\mu=0$.
+
+%This is an example chapter. 
```

Comment:
Why is this chapter commented out?

Reply:
It's a chapter from the original template, should be removed.